controllers/controllers.js

```diff
@@ -86,4 +87,29 @@ exports.postCommentById = (req, res, next) => {
     } else {
         throw { status: 400, msg: "Bad Request" };
     }
+}
+
+exports.patchReview = (req, res, next) => {
+    const id = req.params.id;
+    const votes = req.body.inc_votes;
+    reviewID(id)
+    .then(idTrue => {
+        let regex = RegExp(/^\W?[0-9]+$/)
+        if (validateKeys(req.body, ["inc_votes"]) === false) {
```

Comment:
I like this approach of separating the validation to validation functions. Nice work